openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2190,9 +2190,11 @@ The definitions in (#authorization-ldp-vc) apply for Credentials of this type as
 
 The definitions in (#credential-response-jwt-vc-json) apply for Credentials of this type as well.
 
-## ISO mDL
+## Mobile Documents or mdocs (ISO/IEC 18013 and ISO/IEC 23220 series) {#mdocs}
 
-This section defines a Credential Format Profile for Credentials complying with [@!ISO.18013-5].
+This section defines a Credential Format Profile for credentials that conform to the mobile document (mdoc) format specified in ISO/IEC 18013-5 [@ISO.18013-5] and the ISO/IEC 23220 series [@ISO.23220-1].
```

Comment:
Not sure why the GitHub checks aren't running for this MR, but this surely can't compile now that the definition of this reference has been removed?

Reply:
(I think the github checks aren't running because there's conflicts.)

Reply:
I removed all references of 23220.